org.corluma.Corluma.yml

```diff
@@ -0,0 +1,16 @@
+app-id: org.corluma.Corluma
```

Comment:
the domain `corluma.org` doesn't exist. This app-id does  not meet the requirements.

Reply:
fixed to point to net.corluma instead.